binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -128,6 +130,14 @@ public void run() {
                                 }
                             }
                         }
+
+                        final Config config = fullConfig.getConfig();
+                        if (config != null) {
+                            //TODO persist changes if ticket# is available
```

Comment:
Could you please add the concrete ticket with the link number here:  https://bugs.eclipse.org/bugs/show_bug.cgi?id=461874